yt_dlp/extractor/nuum.py

```diff
@@ -0,0 +1,133 @@
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    int_or_none,
+    parse_iso8601,
+    traverse_obj,
+    try_get,
+)
+
+
+class NuumBaseIE(InfoExtractor):
+
+    def _fetch(self, path, video_id, description, query={}):
```

Comment:
`_call_api` is the convention
```suggestion
    def _call_api(self, path, video_id, description, query={}):
```